config.yml.erb

```diff
@@ -583,3 +583,7 @@ cdo_amplitude_api_key:
 
 # API Key for doublethedonation.com
 ddconf_api_key: !Secret
+
+# Credentials for OpenAI API
+openai_api_chat_key: !Secret
```

Comment:
Should this just be `openai_api_key`? I think this key would vary per-environment rather than per-application.

Reply:
Not a strong feeling, but I think it might be nice to have the keys segmented by application. I know they don't allow splitting metrics by key yet, but I'd be slightly surprised if that's not on their roadmap, and we'd probably (?) want the keys separate for each internal app at that point.